tests/test_main.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import json
+from main import app
+from fastapi.testclient import TestClient
+
+client = TestClient(app)
+
+
+def test_there_is_data_in_json_file():
+    with open('events.json', 'r') as file:
+        data = json.load(file)
+    assert len(data) > 0
+
+
+def test_get_event_with_valid_customer_id():
```

Comment:
This unit test has a great structure however it's an issue that the test is not isolated. If the file doesn't exist or someone modifies this data the test will start failing. In this very specific scenario, you could take a look at mock_open and patch the file that's actually being used and replace it to return fixed testing data.